src/x264_subscriber.cpp

```diff
@@ -92,18 +92,20 @@ namespace x264_image_transport {
 		    ROS_ERROR("Cannot allocate frame");
 		    return;
 		}
-		
+
 		ROS_INFO("Making buffer %d", i);
 		// Determine required buffer size and allocate buffer
-		int numBytes=avpicture_get_size(PIX_FMT_RGB24, codecs[i].m_pCodecCtx->width,codecs[i].m_pCodecCtx->height);
+		int numBytes=avpicture_get_size(AV_PIX_FMT_RGB24, codecs[i].m_pCodecCtx->width,codecs[i].m_pCodecCtx->height);
 		codecs[i].m_buffer=(uint8_t *)av_malloc(numBytes*sizeof(uint8_t));
 
 		// Assign appropriate parts of buffer to image planes in pFrameRGB
 		// Note that pFrameRGB is an AVFrame, but AVFrame is a superset
 		// of AVPicture
-		avpicture_fill((AVPicture *)codecs[i].m_pFrameRGB, codecs[i].m_buffer, PIX_FMT_RGB24,
+		avpicture_fill((AVPicture *)codecs[i].m_pFrameRGB, codecs[i].m_buffer, AV_PIX_FMT_RGB24,
 		               codecs[i].m_pCodecCtx->width, codecs[i].m_pCodecCtx->height);
 		codecs[i].m_img_convert_ctx = NULL;
+
+        av_log_set_level(AV_LOG_QUIET); // silence all the errors/warnings
```

Comment:
spacing here looks wrong. I think you need to use tabs. Since the rest of the file uses tabs just match that. Normally in our code we use spaces and never tabs.